compiler/rustc_hir_analysis/src/check/check.rs

```diff
@@ -518,6 +518,9 @@ fn check_item_type(tcx: TyCtxt<'_>, id: hir::ItemId) {
             check_static_inhabited(tcx, id.owner_id.def_id);
             check_static_linkage(tcx, id.owner_id.def_id);
         }
+        // FIXME(generic_consts): Do we want to enforce that type params are used? I don't think so.
```

Comment:
Given that they don't have variance, it shouldn't matter.

Reply:
Params shouldn't need to be used, and yeah, consts have no variance.